starters/tailwind/src/AlertDialog.tsx

```diff
@@ -1,3 +1,4 @@
+'use client';
```

Comment:
added `'use client'` to all starter kit examples, which is how users should generally implement our components as well.